liquidity-module/index.md

```diff
@@ -23,28 +28,38 @@ In this tutorial, you create your own blockchain, send tokens to another blockch
 - Create a pool with your token
 - Use the pool with your token
 
-## Prerequisites
+
+## Requirements
```

Comment:
for some reason (teach me if you know) the Prerequisites heading does not appear on the published pages... let's try another heading (in case Prerequisites is a reserved word or something?) anyhooo